src/Controls/tests/Core.UnitTests/LabelTests.cs

```diff
@@ -178,6 +178,15 @@ public void EntryCellYAlignBindingMatchesVerticalTextAlignmentBinding()
 			Assert.Equal(TextAlignment.End, labelVerticalTextAlignment.VerticalTextAlignment);
 		}
 
+		[Fact]
+		public void OriginalLabelTextNotUpdatingAfterBindingIsSet()
+		{
```

Comment:
The test was added to validate that this scenario fails?.
<img width="1116" alt="Captura de pantalla 2023-08-03 a las 16 25 15" src="https://github.com/dotnet/maui/assets/6755973/1b85b60b-f820-4899-ad0b-98db8810bfd8">